olp-cpp-sdk-core/include/olp/core/cache/KeyValueCache.h

```diff
@@ -121,6 +121,14 @@ class CORE_API KeyValueCache {
    * @return True if the values are removed; false otherwise.
    */
   virtual bool RemoveKeysWithPrefix(const std::string& prefix) = 0;
+  /**
+   * @brief Check if key is in the cache.
+   *
+   * @param key The key for the value.
+   *
+   * @return True if the key/value cached ; false otherwise.
+   */
+  virtual bool Exist(const std::string& key) = 0;
```

Comment:
Lets call it Contains().
NOTE: This is a breaking change! This means all user code will not compile as there is a new abstract API that is not overridden. You need to provide a default impl to not break API.

```suggestion
  virtual bool Contains(const std::string& key) {
      CORE_UNUSED(key);
   };
```